### PR TITLE
Add multi-links extension

### PR DESCRIPTION
**Pull request by wrick17**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/multi-links/src/multi-links.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { getSelectedText, open, showHUD } from "@raycast/api";
+import extractUrls from "extract-urls";
+
+export default async function main() {
+  try {
+    const selectedText = await getSelectedText();
+    const urls = extractUrls(selectedText);
+    if (urls.length > 0) {
+      for (const url of urls) {
+        await open(url, "company.thebrowser.Browser");
```
Comment (pernielsentikaer) [inline extensions/multi-links/src/multi-links.ts:10]: Why not open it in the default browser instead?

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by wrick17** [comment]
> Hi 👋 
> 
> 
> 
> Thanks for your contribution 💪 
> 
> 
> 
> I have now tested your extension, and I have some feedback ready for you:
> 
> 
> 
> I'm looking forward to testing this extension again 🔥 
> 
> 
> 
> Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

Hi @pernielsentikaer that's a great suggestion. I couldn't find a proper way to open the default browser so I used this code I found somewhere on the internet. It does open my default browser, however I've not tested it extensively. If you tell me what's the correct way to open it in the default browser, I'll gladly put that in.

**Comment by pernielsentikaer** [comment]
`await open(url);` should be more than enough 🙂

**Comment by wrick17** [comment]
> `await open(url);` should be more than enough 🙂

Ah! That simple huh? 😅

**Comment by wrick17** [comment]
> `await open(url);` should be more than enough 🙂

Done!

File: extensions/multi-links/src/multi-links.ts (hunk 1 above)
Comment (wrick17) [reply extensions/multi-links/src/multi-links.ts]: Done!

**Comment by pernielsentikaer** [comment]
I tried to get it to work, if I debug selectedText and urls which gives me

```
23:04:53.286 🚀 ~ main ~ selectedText: Hello www.raycat.com is really good www.dr.dk
23:04:53.286 🚀 ~ main ~ urls: undefined
```

Shouldn't it give some urls?

**Comment by wrick17** [comment]
It particularly looks for http or https to make sure that they are links
and not things like package names and such.

**Comment by wrick17** [comment]
> Hi 👋 
> 
> 
> 
> Looks good to me, approved 🔥

Awesome. Thanks. I've also linked my Github account.

**Pull request merged**